src/management/moderation/components/moderation-summary/moderation-summary.html

```diff
@@ -18,7 +18,9 @@ <h2 ng-bind="moc.offer.name"></h2>
                 <udb-calendar-summary offer="::moc.offer" show-opening-hours="true"></udb-calendar-summary>
             </p>
 
-            <div class="content" ng-bind-html="moc.offer.description"></div>
+            <div class="content"
```

Comment:
If I look at [the structure](https://github.com/cultuurnet/udb3-angular/blob/dbb708fe7d7cd32bb01340941dfb59b66ad83b11/src/management/moderation/moderation.service.spec.js#L38-L43) of `moc.offer.description`it returns an object that is keyed by a language. 

Looks like that is supposed to be translated by the [language filter](https://github.com/cultuurnet/udb3-angular/blob/4fc4750b0e4d00e9c5075dac8c6b89d38f829a77/src/search/filters/json-ld-lang.filter.js). 

https://github.com/cultuurnet/udb3-angular/blob/dbb708fe7d7cd32bb01340941dfb59b66ad83b11/src/management/moderation/components/moderation-offer/moderation-offer.component.js#L46

Maybe double-check if that's being done correctly?